rte/kernels-openacc/mo_rte_solver_kernels.F90

```diff
@@ -1162,21 +1158,20 @@ subroutine apply_BC_0(ncol, nlay, ngpt, top_at_1, flux_dn) bind (C, name="apply_
 
     !   Upper boundary condition
     if(top_at_1) then
-      !$acc  parallel loop collapse(2) async
+      !$acc  parallel loop collapse(2)
       do igpt = 1, ngpt
         do icol = 1, ncol
           flux_dn(icol,      1, igpt)  = 0._wp
         end do
       end do
     else
-      !$acc  parallel loop collapse(2) async
```

Comment:
Are you removing asyncs because of correctness issues? I have a feeling we're going to want these back in the code when we run with small per-node workloads.

Reply:
And we'll probably eventually need to add them to other parts of the code as well to hide kernel launch overheads

Reply:
This was the only place we had async. I'm happy to add it back when I understand it better and can apply it uniformly.